x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/helpers.ts

```diff
@@ -36,6 +36,8 @@ export const formatRules = (rules: Rule[], selectedIds?: string[]): TableData[]
     },
     tags: rule.tags ?? [],
     activate: rule.enabled,
+    status: rule.status == null ? null : rule.status,
+    statusDate: rule.status_date == null ? null : rule.status_date,
```

Comment:
I think you can use your same technique as you have below:

```ts
status: rule.status ?? null
statusCode: rule.status_date ?? null
```

Would look nicer.